lib/pre_assembly.rb

```diff
@@ -35,6 +36,7 @@ module PreAssembly
   def self.retry_handler(method_name, logger, params = {})
     Proc.new do |exception, attempt_number, total_delay|
       logger.send "      ** #{method_name} FAILED **; with params of #{params.inspect}; and trying attempt #{attempt_number} of #{Dor::Config.dor.num_attempts}; delayed #{Dor::Config.dor.total_delay} seconds"
+      Honeybadger.notify(exception)
```

Comment:
Are we sure we want to notifiy honeybadger on every retry?

Reply:
no, that would not be a good idea, i'll remove this line